website/content/api-docs/agent/index.mdx

```diff
@@ -720,7 +721,9 @@ The corresponding CLI command is [`consul force-leave`](/commands/force-leave).
 
 ### Parameters
 
-- `node` `(string: <required>)` - Specifies the name of the node to be forced into `left` state. This is specified as part of the URL.
+- `node` `(string: <required>)` - Specifies the name of the node to be forced into `left` state.
+  This is specified as part of the URL as a path segment and
```

Comment:
I tried defining this sentence in a partial so it could be defined in one place and referenced everywhere else, but that doesn't seem to work. You can't include a partial inline, so the sentence couldn't be injected as a part of the parameter description itself (it would show up as a separate sentence underneath, unindented).

Reply:
Not that I'm aware of. I think I only discovered this feature by stumbling across its usage in a few pages and then looking into how it worked. I don't think there's any guidance in the `website/README.md` file.

Reply:
I can see us backing ourselves into an undesirable position without developing some ground rules. At the very least, we should include a comment describing known sections where the partial has been used. 

Ideally, we would leverage this ability to create fully modular docs that could be assembled into different outputs. If we had conditional processing feature, then we could tag some blocks as "enterprise" or "Kubernetes", for example,  and generate custom sites for different experiences. We would probably need a CMS layer . . . but I digress.